lib/DDG/Spice/Untappd.pm

```diff
@@ -0,0 +1,38 @@
+package DDG::Spice::Untappd;
+
+# ABSTRACT: Discover your favorite beers
+
+use DDG::Spice;
+use Text::Trim;
+
+spice is_cached => 1;
+spice proxy_cache_valid => "200 1d";
+
+spice wrap_jsonp_callback => 1;
+
+spice to => 'https://api.untappd.com/v4/search/beer?' .
+                'client_id={{ENV{DDG_SPICE_UNTAPPD_CLIENT_ID}}}&' .
+                'client_secret={{ENV{DDG_SPICE_UNTAPPD_CLIENT_SECRET}}}&' .
+                'q=$1';
+
+triggers any => 'untappd';
+triggers end => qw/
+    ale apa ipa aipa dipa bipa porter stout bitter barleywine
+    saison lambik dubbel tripel quadrupel framboise gueuze kriek witbier
+    altbier weissbier dunkelweizen gose
+    hefeweizen kristalweizen roggenbier weizenbock
+    weissbier marzen dunkel weizenbier
+    bock doppelbock rauchbier schwarzbier
+    maibock eisbock kellerbier kolsch zoigl
+    lager pilsner
+    happoshu
+    sahti
+/;
+
+handle query_lc => sub {
+    $_ =~ s/\buntappd\b//g;
+    $_ =~ s/\s\s/ /g;
```

Comment:
Without this line, [test case](https://github.com/duckduckgo/zeroclickinfo-spice/pull/2746/files#diff-2d7ba98ad82fad36b03ee5caacc68e0aR37) will be failed.

```
#   Failed test 'Testing query tratata uuntappdd untappd untappdd'
#   at /home/vagrant/perl5/lib/perl5/DDG/Test/Spice.pm line 53.
#     Structures begin differing at:
#          $got->{call} = '/js/spice/untappd/tratata%20uuntappdd%20%20untappdd'
#     $expected->{call} = '/js/spice/untappd/tratata%20uuntappdd%20untappdd'
```